config/config.go

```diff
@@ -318,9 +327,35 @@ func validate() error {
 		}
 	}
 
+	if vip.GetString(OperatorExtraDomainKey) != "" ||
+		vip.GetString(OperatorExtraIPKey) != "" {
+		if vip.GetString(ConnectUrlHost) != "" {
+			return fmt.Errorf("%s and %s/%s cannot be set together",
+				ConnectUrlHost, OperatorExtraDomainKey, OperatorExtraIPKey)
+		}
+	}
+
 	return nil
 }
 
+func GetHost() string {
+	host := GetString(ConnectUrlHost)
+
+	if vip.GetString(OperatorExtraIPKey) != "" {
+		host = vip.GetString(OperatorExtraIPKey)
+	}
+
+	if vip.GetString(OperatorExtraDomainKey) != "" {
+		host = vip.GetString(OperatorExtraDomainKey)
+	}
+
+	if host == "" {
+		host = "localhost"
+	}
+
+	return host
+}
```

Comment:
assuming we pass [this](https://github.com/tdex-network/tdex-daemon/pull/625/files#r955216201) check we should give priority to ConnectUrlHost, if not given domain should have prioirity over ip (if both given).

Reply:
Done